src/main/kotlin/fi/metatavu/timebank/api/persistence/repositories/VacationsRepository.kt

```diff
@@ -0,0 +1,84 @@
+package fi.metatavu.timebank.api.persistence.repositories
+
+import fi.metatavu.timebank.api.persistence.model.VacationRequest
+import io.quarkus.hibernate.reactive.panache.Panache
+import io.quarkus.hibernate.reactive.panache.PanacheRepositoryBase
+import io.quarkus.panache.common.Parameters
+import io.smallrye.mutiny.coroutines.awaitSuspending
+import java.time.LocalDate
+import java.util.UUID
+import javax.enterprise.context.ApplicationScoped
+
+/**
+ * Manages VacationRequest JPA entity
```

Comment:
As stated in some earlier comment, change to consistent naming e.g. `VacationRequestsRepository`.